Python/lipd/download_lipd.py

```diff
@@ -20,38 +17,40 @@ def get_download_path():
     Determine the OS and the associated download folder.
     :return str Download path:
     """
-    if os.name == 'nt':
+    if os.name == "nt":
         import winreg
-        sub_key = r'SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders'
-        downloads_guid = '{374DE290-123F-4565-9164-39C4925E467B}'
+
+        sub_key = r"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders"
+        downloads_guid = "{374DE290-123F-4565-9164-39C4925E467B}"
         with winreg.OpenKey(winreg.HKEY_CURRENT_USER, sub_key) as key:
             location = winreg.QueryValueEx(key, downloads_guid)[0]
         return location
     else:
-        return os.path.join(os.path.expanduser('~'), 'Downloads')
+        return os.path.join(os.path.expanduser("~"), "Downloads")
 
 
 def download_from_url(src_url):
     """
     Use the given URL and destination to download and save a file
     :param str src_url: Direct URL to lipd file download
-    :param str dst_path: Local path to download file to, including filename and ext. ex. /path/to/filename.lpd
+    :param str dst_path: Local path to download file to, including filename and ext.
+        ex. /path/to/filename.lpd
     :return none:
     """
     if "MD982181" not in src_url:
         try:
             # Find the user's local downloads folder, os dependent
-            #dir_downloads = get_downloads_folder()
+            # dir_downloads = get_downloads_folder()
```

Comment:
should be a little careful not to confuse commented-out code (likely should be deleted) with real comments